tui2/pages/output/output.go

```diff
@@ -72,6 +72,16 @@ func New(c common.Common, config *request.Config) (*Output, error) {
 	// 	return nil, err
 	// }
 
+	getBytesEncodingPerNetwork := func(endpoint string) dynamic.BytesRepresentation {
+		if manifest.HardcodedEndpoints["solana-mainnet-beta"] == endpoint {
+			return dynamic.BytesAsBase58
+		}
+
+		return dynamic.BytesAsBase64
```

Comment:
Can you default to hex instead, seems like a better default for most chains for now. 

The network registry from Pinax was planned to be integrated which should come with bytes encoding.